src/optimise/optimisers.jl

```diff
@@ -133,16 +133,17 @@ opt = RMSProp(0.002, 0.95)
 mutable struct RMSProp <: AbstractOptimiser
   eta::Float64
   rho::Float64
+  eps::Float64
   acc::IdDict
 end
 
-RMSProp(η = 0.001, ρ = 0.9) = RMSProp(η, ρ, IdDict())
+RMSProp(η = 0.001, ρ = 0.9, ϵ = ϵ) = RMSProp(η, ρ, ϵ, IdDict())
```

Comment:
This is what we use in Optimisers.jl.

```suggestion
RMSProp(η = 0.001, ρ = 0.9, ϵ = eps(typeof(η))) = RMSProp(η, ρ, ϵ, IdDict())
```

Reply:
I didn't want to change the value of `eps` Flux is currently using, so as to not change behavior.
So I continue to use to the value of `1e-8` here.
But I did use the type of eta as you suggest.